app/src/main/java/org/simple/clinic/bloodsugar/BloodSugarReading.kt

```diff
@@ -42,6 +43,21 @@ data class BloodSugarReading(val value: String, val type: BloodSugarMeasurementT
       }
     }
 
+  val displayType: Int
+    get() = when (type) {
+      Random -> R.string.bloodsugar_reading_type_rbs
+      PostPrandial -> R.string.bloodsugar_reading_type_ppbs
+      Fasting -> R.string.bloodsugar_reading_type_fbs
+      HbA1c -> R.string.bloodsugar_reading_type_hba1c
+      else -> throw IllegalArgumentException("Unknown blood sugar type $type")
+    }
+
+  val displayUnit: Int
+    get() = when (type) {
+      Random, PostPrandial, Fasting, is Unknown -> R.string.bloodsugar_reading_unit_type_mg_dl
+      HbA1c -> R.string.bloodsugar_reading_unit_type_percentage
+    }
```

Comment:
Can we add `@StringRes` annotations to the properties?

Reply:
Done